src/components/tab-bar/tab-bar.tsx

```diff
@@ -0,0 +1,139 @@
+import { Component, h, Prop, Element, State } from '@stencil/core';
+import {
+    MDCTabBar,
+    MDCTabBarActivatedEvent
+} from '@limetech/mdc-tab-bar';
+import { strings } from '@limetech/mdc-tab-bar/constants';
+import { Tab } from './tab.types';
+const { TAB_ACTIVATED_EVENT } = strings;
+
+@Component({
+    tag: 'limel-tab-bar',
+    styleUrl: 'tab-bar.scss',
+    shadow: true
+})
+export class TabBar {
+
+    @Element()
+    private element: HTMLElement;
+
+    /**
+     * List of tabs to display
+     */
+    @Prop()
+    public tabs: Tab[];
+
+    private mdcTabBar: MDCTabBar;
+
+    @State()
+    private selectedTab = this.tabs[0];
+
+    constructor() {
+        this.handleAction = this.handleAction.bind(this);
+
+    }
+
+    public componentDidLoad() {
+        this.element.focus();
+        this.setup();
+    }
+
+    private setup() {
+        this.element.focus();
+        const element = this.element.shadowRoot.querySelector('.mdc-tab-bar');
+        if (!element) {
+            return;
+        }
+
+        this.mdcTabBar = new MDCTabBar(element);
+        (this.mdcTabBar as any).foundation_.adapter_.getFocusedTabIndex = () => {
+            var tabElements = this.getTabElements();
+            var activeElement = this.element.shadowRoot.activeElement;
+            return tabElements.indexOf(activeElement);
+        };
+
+        this.setupListeners();
+    }
+
+    private getTabElements() {
+        return [].slice.call(this.element.shadowRoot.querySelectorAll('.mdc-tab'));
+    }
+
+    private setupListeners() {
+        this.mdcTabBar.listen(TAB_ACTIVATED_EVENT, this.handleAction);
+    }
+
+    private handleAction(event: MDCTabBarActivatedEvent) {
```

Comment:
We should let consumers know that another tab has been selected by emitting an event from this component